ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -10,7 +10,8 @@ STS=${STS:-true}
 HOSTED_CP=${HOSTED_CP:-false}
 COMPUTE_MACHINE_TYPE=${COMPUTE_MACHINE_TYPE:-"m5.xlarge"}
 OPENSHIFT_VERSION=${OPENSHIFT_VERSION:-}
-CHANNEL_GROUP=${CHANNEL_GROUP}
+TARGET_VERSION="${VERSION}"
```

Comment:
Why do you need a new variable `TARGET_VERSION` here?  Can't you just use `VERSION`?